common/templates/options.php

```diff
@@ -63,7 +65,7 @@
                             <?php _e('Yes, pop-up after purchase at "thank you" page. Privacy first review option: order data will only be shared after permission from the customer.', 'webwinkelkeur'); ?>
                         </label><br>
                         <label>
-                            <input type="radio" name="<?= $plugin->getOptionName('invite');?>" value="<?= WooCommerce::AFTER_EVERY_ORDER ?>" <?= $config['invite'] == WooCommerce::AFTER_EVERY_ORDER || !$config['invite'] ? 'checked' : ''; ?> />
+                            <input type="radio" name="<?= $plugin->getOptionName('invite');?>" value="<?= WooCommerce::AFTER_EVERY_ORDER ?>" <?= $config['invite'] == WooCommerce::AFTER_EVERY_ORDER || $config['invite'] == '' ? 'checked' : ''; ?> />
```

Comment:
This should work fine, but you can actually solve the problem much easier by adding the `invite` key to the `BasePlugin::getDefaultConfig` method. Then you can just remove the `|| $config['invite'] == ''` part.